test/fail_compilation/traits.d

```diff
@@ -0,0 +1,24 @@
+/*
+REQUIRED_ARGS:
+PERMUTE_ARGS:
+*/
+
+/************************************************************/
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
+fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
+fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
+fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
+fail_compilation/traits.d(104): Error: expected 0 arguments for `allTargetInfos` but had 1
+---
+*/
+
+#line 100
```

Comment:
Minor detail, I would avoid using `#line`, instead the `AUTO_UPDATE=1` [1] flag can be used to update the `TEST_OUTPUT` section.

[1] https://github.com/dlang/dmd/tree/5dba233b45f639594bafa0138ee936c33210b956/test#automatically-update-the-test_output-segments